src/scss/layout/_title.scss

```diff
@@ -8,14 +8,14 @@
 
   align-self: center;
   grid-area: title;
-  margin: 0;
   position: relative;
 
+  margin: -50px 0 0 0; // needs a breakpoint for smaller screens
+
   #hero + & {
     // clip the horizontal and bottom shadows only
     clip-path: inset(var(--outset) calc(var(--outset) / 1.5) 0 0);
     filter: drop-shadow(0 0 var(--blur) var(--shadow));
-    margin-top: 0;
```

Comment:
Because of the negative margin above, this is breaking pages that have a hero image:
 
![Screenshot 2023-03-08 at 2 04 46 PM](https://user-images.githubusercontent.com/104161/223850216-2acac183-feb2-4657-b9e9-37723f6abc41.jpg)